modules/mauth-proxy/src/main/resources/reference.conf

```diff
@@ -8,4 +8,8 @@ proxy {
 app {
   uuid: ${?APP_MAUTH_UUID}
   private_key: ${?APP_MAUTH_PRIVATE_KEY}
+}
+
+mauth {
+  disable_v1: false
```

Comment:
should this be `enable_v1 : true`?

Reply:
Currently jvm version does the same way as .net version. will re-think this. Thanks